src/server/snapshot.h

```diff
@@ -85,7 +76,7 @@ class SliceSnapshot {
 
   // Waits for a regular, non journal snapshot to finish.
   // Called only for non-replication, backups usecases.
-  void Join() {
+  void WaitSnapshoting() {
```

Comment:
WaitSnapshoting -> `WaitSnapshotting`
please check if you have more `shoting`  cases

Reply:
done